tests/mozdef_util/test_elasticsearch_client.py

```diff
@@ -28,11 +28,11 @@ def setup(self):
     def get_num_events(self):
         self.refresh('events')
         search_query = SearchQuery()
-        search_query.add_must(TermMatch('_type', '_doc'))
+        search_query.add_must(TermMatch('_type', DOCUMENT_TYPE))
```

Comment:
Is this needed since it will always be 1 for a particular index?
Should this be for "type" instead or some other field?

Reply:
This is basically just a utility helper to return the number of events in an index, so that tests can verify that they were able to write events to ES.

Reply:
ahh makes sense then :)